modules/prebidServerBidAdapter/index.js

```diff
@@ -170,12 +171,29 @@ function doAllSyncs(bidders) {
 
   const thisSync = bidders.pop();
   if (thisSync.no_cookie) {
-    doBidderSync(thisSync.usersync.type, thisSync.usersync.url, thisSync.bidder, utils.bind.call(doAllSyncs, null, bidders));
+    doPreBidderSync(thisSync.usersync.type, thisSync.usersync.url, thisSync.bidder, utils.bind.call(doAllSyncs, null, bidders));
   } else {
     doAllSyncs(bidders);
   }
 }
 
+/**
+ * Modify the cookie sync url from prebid server to add new params.
+ *
+ * @param {string} type the type of sync, "image", "redirect", "iframe"
+ * @param {string} url the url to sync
+ * @param {string} bidder name of bidder doing sync for
+ * @param {function} done an exit callback; to signify this pixel has either: finished rendering or something went wrong
+ */
+function doPreBidderSync(type, url, bidder, done) {
+  if (typeof _s2sConfig.syncUrlModifier[bidder] === 'function') {
```

Comment:
Should this be testing for syncUrlModifier before attempting to index into `bidder`?

i.e.
```
if (_s2sConfig.syncUrlModifier and typeof _s2sConfig.syncUrlModifier[bidder] === 'function')
```